src/Avalonia.Diagnostics/ViewModels/PropertyDetails.cs

```diff
@@ -41,19 +38,19 @@ public PropertyDetails(AvaloniaObject o, AvaloniaProperty property)
         public string Priority
         {
             get { return _priority; }
-            private set { this.RaiseAndSetIfChanged(ref _priority, value); }
+            private set { _priority = value; RaisePropertyChanged(); }
```

Comment:
It's probably better to implement `RaiseAndSetIfChanged<T>(ref T prop, T value, [CallerMemberName]string name =null)` instead